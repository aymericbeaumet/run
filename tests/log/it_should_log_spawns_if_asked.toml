[[run]]
cmd = ["coreutils", "echo", "hey!"]