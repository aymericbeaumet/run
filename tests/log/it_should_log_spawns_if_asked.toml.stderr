[coreutils] coreutils echo hey! spawned
[coreutils] coreutils echo hey! terminated with status code 0